### DISCUSSION TITLE
Bl tuch with octoprint

### DISCUSSION THREAD

**Discussion by tonyG433** [Jyers/Marlin#704] [Q&A] [answered]
What m code should i type in the bed mesh visualizer plugin? I saw a guy using M420 T1 V, but I am not sure.

**Comment by tonyG433** [comment]
also im using Bl touch 3x3 4.2.7

**Comment by Jyers** [comment, marked as answer]
M420 V1 should be all you need

**Reply by tonyG433** [reply]
thanks, will try it out
